big_tests/tests/inbox_extensions_SUITE.erl

```diff
@@ -25,7 +25,10 @@
          end_per_testcase/2]).
 
 all() ->
-    inbox_helper:skip_or_run_inbox_tests(tests()).
+    Tests = tests(),
+    distributed_helper:temporary_allow_nodes(fun() ->
+            inbox_helper:skip_or_run_inbox_tests(Tests)
+        end, [mim]).
```

Comment:
I'm not sure to understand when am I supposed to use `temporary_allow_nodes/1` 🤔

Reply:
in code which is called from `groups/0` CT callback. Or from `all/0` callback.
I.e. almost never. 

Unless you call something like an RPC to a node to figure out which tests to run... like here.

Reply:
Technically, this part of code smell a bit.
Maybe we need a helper module, which would have something like:
```
preset_info() -> #{rdbms => true, elasticsearch => true, is_mssql => false}.
```
and so on.

And we can actually decide if we wanna do an RPC or put these `test.config` presets.

By the way, we don't need to do RPC at all:
```
    {odbc_mssql_mnesia,
     [{dbs, [redis, mssql]},
```

`test.config` has a list of databases, that are supposed to be running during a preset.

Could refactor calls in `all/0` and `groups/0`. But sounds like a separate PR.
So, after that `temporary_allow_nodes` workaround could be removed at all.

 + during that task we can check that testname autocomplition works again (because if you do RPCs, it does not. And that is why there is also `is_ct_running` call usually, but for example, it is missing here too).
 
 Just grabbing DB type from the running node is the most "obvious" solution usually.

Reply:
Also, funny, we get preset inside `Config`. But not in `groups/0` or `all/0` callbacks. 
So, we would have to pass preset as OS env variable probably.